plugins/contiv/host.go

```diff
@@ -261,6 +261,13 @@ func (s *remoteCNIserver) routeToOtherHostStack(hostID uint8, nextHopIP string)
 	return s.routeToOtherHostNetworks(hostNw, nextHopIP)
 }
 
+func (s *remoteCNIserver) routeToOtherManagementIP(managementIP string, nextHopIP string) *vpp_l3.StaticRoutes_Route {
+	return &vpp_l3.StaticRoutes_Route{
+		DstIpAddr:   managementIP + "/32",
+		NextHopAddr: nextHopIP,
+	}
```

Comment:
We will need to make the IP address generic IPv6/IPv4; we'll mrege for now, but IP addressing needs to be fixed overall.